.bumpversion.toml

```diff
@@ -22,3 +22,6 @@ filename = "gen/node/package.json"
 
 [[tool.bumpversion.files]]
 filename = "gen/haskell/utxorpc.cabal"
+
+[[tool.bumpversion.files]]
+filename = "gen/dotnet/Utxorpc.V1.csproj"
```

Comment:
can we name the project `Utxorpc.Spec` instead of `Utxorpc.V1`?
This project will contain more than one version of the interface.